lib/batalert.rb

```diff
@@ -17,19 +18,21 @@ def main
       status = File.open('/sys/class/power_supply/BAT0/status', &:readline)
       status = status.chomp
 
+      volume = Volumerb.value
+
       if capacity < 10 && status == 'Discharging'
         notify = Libnotify.new(summary: 'PUT ON CHARGING, YOUR BATTERY IS AT ' \
-                               "#{capacity}%.", timeout: 3, urgency: :critical)
+                                        "#{capacity}%.", timeout: 3, urgency: :critical)
         notify.update
-        speech = ESpeak::Speech.new("put on charging, your battery is at #{capacity}%.")
+        speech = ESpeak::Speech.new("put on charging, your battery is at #{capacity}%.", amplitude: volume.to_s)
```

Comment:
Not sure why we need to give `volume` as a String, would suggest just passing it as an Integer only.
```suggestion
        speech = ESpeak::Speech.new("put on charging, your battery is at #{capacity}%.", amplitude: volume)
```

Reply:
Bah! You're right. It shouldn't be a string. RuboCop autocorrected stuff for me and I got all confused later. Apologies. Fixed.